lib/image_utils_isolate.dart

```diff
@@ -52,15 +52,15 @@ class ImageUtilsIsolate {
     }
 
     if (image != null) {
-      if (Platform.isIOS) {
-        // ios, default camera image is portrait view
-        // rotate 270 to the view that top is on the left, bottom is on the right
-        // image ^4.0.17 error here
-        image = copyRotate(image, angle: 270);
-      }
-      if (Platform.isAndroid) {
-        image = copyRotate(image, angle: 90);
-      }
+      // if (Platform.isIOS) {
+      //   // ios, default camera image is portrait view
+      //   // rotate 270 to the view that top is on the left, bottom is on the right
+      //   // image ^4.0.17 error here
+      //   image = copyRotate(image, angle: 270);
```

Comment:
Looks good, but Having the rotation here was because of using the isolate to avoid frame drops on the main isolate, and to avoid code duplicate, so can you re add it and pass the rotation arguments to it same as you did in the other code 

And of course remove the rotation from the main classes 

I think then it will be good to go